lib/pure/collections/sequtils.nim

```diff
@@ -1096,8 +1096,10 @@ macro mapLiterals*(constructor, op: untyped;
   ## be used to map every array element to some target type:
   runnableExamples:
     let x = mapLiterals([0.1, 1.2, 2.3, 3.4], int)
-    doAssert x is array[4, int]
-    doAssert x == [int(0.1), int(1.2), int(2.3), int(3.4)]
+    assert x is array[4, int]
+    assert x == [int(0.1), int(1.2), int(2.3), int(3.4)]
```

Comment:
IMO we should at least mention an example like this:
```nim
let x = mapLiterals([(1, 2), (3, 4)], int8)
assert x.type == ...
assert x == ...
```
which at least shows some usefulness for this feature (unlike the other examples given)

Reply:
what about this one?

Reply:
Better to add more examples on non-Deprecated proc that are not footguns. 😆